src/js/components/DataTable/DataTable.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
Do we still need to disable this? I skimmed the file and didn't see any nested ternaries.

Reply:
Nope. Good catch